include/hw_unique_key.h

```diff
@@ -22,34 +22,20 @@ extern "C" {
 
 #include <stdbool.h>
 
-#ifdef __ZEPHYR__
+#if defined(__NRF_TFM__)
 #include <autoconf.h>
-#ifdef CONFIG_HAS_HW_NRF_KMU
-	#define HUK_HAS_KMU
-#endif
-#ifdef CONFIG_HAS_HW_NRF_CC310
-	#define HUK_HAS_CC310
-#endif
-#ifdef CONFIG_HAS_HW_NRF_CC312
-	#define HUK_HAS_CC312
 #endif
 
-#else /* __ZEPHYR__ */
-#define HUK_HAS_KMU
-#ifdef NRF9160_XXAA
-	#define HUK_HAS_CC310
-#endif
-#ifdef NRF5340_XXAA_APPLICATION
-	#define HUK_HAS_CC312
-#endif
-#endif /* __ZEPHYR__ */
+#define HUK_HAS_KMU   CONFIG_HW_HAS_NRF_KMU
+#define HUK_HAS_CC310 CONFIG_HW_HAS_NRF_CC310
+#define HUK_HAS_CC312 CONFIG_HW_HAS_NRF_CC312
 
-#ifdef HUK_HAS_CC310
+#if defined(HUK_HAS_CC310)
```

Comment:
I'm not sure this will work now. Won't it always be defined?

Reply:
You're right. Will fix.